src/ontology/mondo-edit.obo

```diff
@@ -258187,14 +258202,16 @@ xref: Orphanet:588 {source="MONDO:subClassOf", source="OMIM:613150"}
 xref: Orphanet:899 {source="MONDO:subClassOf", source="MONDO:relatedTo", source="OMIM:613150"}
 xref: UMLS:C3150411 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo", source="NCIT:C126742", source="OMIM:613150"}
 is_a: MONDO:0000171 {source="OMIM:613150", source="ORDO:899/btnt"} ! muscular dystrophy-dystroglycanopathy, type A
-is_a: MONDO:0018939 ! muscle-eye-brain disease
+is_a: MONDO:0018939 {source="https://orcid.org/0000-0002-4142-7153"} ! muscle-eye-brain disease
 is_a: MONDO:0020121 {source="MONDO:Redundant", source="NCIT:C126742", source="OMIM:613150/inferred"} ! muscular dystrophy
+is_a: MONDO:0700071 {source="https://clinicalgenome.org/affiliation/50061/"} ! myopathy caused by mutation in POMT2
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/19743 {source="mim2gene_medgen"} ! POMT2
 property_value: confidence "0.5294117647058822" xsd:double
 
 [Term]
 id: MONDO:0013155
 name: muscular dystrophy-dystroglycanopathy (congenital with intellectual disability), type B3
+synonym: "congenital muscular dystrophy-POMGNT1 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/